src/css/guide/index.css

```diff
@@ -1,28 +1,26 @@
 @import './sidebar.css';
 @import './content.css';
+@import './header.css';
 
 main {
-	/*overflow: scroll;
+	overflow: scroll;
 	height: calc(100vh - 4em);
 	position: absolute;
 	width: 100%;
-	top: 4em;*/
+	top: 4em;
 }
 
 header {
-	/*position: fixed;*/
+	position: fixed;
 	width: 100%;
 	left: 0;
 	top: 0;
-	z-index: 2;
+	z-index: 200;
 }
 
-@media (min-width: 768px) {
+@media (max-width: 768px) {
 	main {
-		overflow: scroll;
-		height: calc(100vh - 4em);
-		position: absolute;
-		width: 100%;
-		top: 4em;
+		height: calc(100vh - 3em);
```

Comment:
Could you give a rough estimate in % in the line before for browsers, who don't understand calc?